{"name": "test_get_users", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 200\n +  where 401 = <Response [401]>.status_code", "trace": "def test_get_users():\n        response = requests.get(\"https://reqres.in/api/users?page=2\")\n>       assert response.status_code == 200\nE       assert 401 == 200\nE        +  where 401 = <Response [401]>.status_code\n\ntests\\api\\test_users_api.py:5: AssertionError"}, "start": 1759853509977, "stop": 1759853511250, "uuid": "501ef24d-164e-4601-826c-2a4f2e2c2e65", "historyId": "4579b9adc2e4fc433582bee98c930d39", "testCaseId": "4579b9adc2e4fc433582bee98c930d39", "fullName": "tests.api.test_users_api#test_get_users", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_users_api"}, {"name": "host", "value": "Shawns-Lenovo"}, {"name": "thread", "value": "6788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_users_api"}], "titlePath": ["tests", "api", "test_users_api.py"]}